src/mon/PGMap.cc

```diff
@@ -925,7 +925,11 @@ void PGMapDigest::dump_object_stat_sum(
     if (verbose) {
       f->dump_int("quota_objects", pool->quota_max_objects);
       f->dump_int("quota_bytes", pool->quota_max_bytes);
-      f->dump_int("dirty", sum.num_objects_dirty);
+      if (pool->is_tier()) {
+        f->dump_int("dirty", sum.num_objects_dirty);
+      } else {
+	f->dump_int("dirty", 0);
```

Comment:
would be better if we could be more consistent at using tab versus space for indent?

Reply:
Time for a c++ linter?

Reply:
```py
time = when(any(hero.is_willing_to(create(a pre-commit hook) or
                                   create(a jenkins job using something like clang-format))
                for hero in us))
```

Reply:
Hehehe, I think that the main thing that divert us from doing this is the amount of work required to clean up everything... If we could find a way to only fail on the modified code, that'd probably be more encouraging... Do you have any clue on how to do that?

Reply:
something like a script which

1. runs clang-format on all changed files 
2. pipes the output to a filter which checks if the line number falls into the area defined by `git diff`

if there is any stuff shows at the other end of the pipe, the author of the change should be alerted.